src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,198 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.27.0.dev0 (3/14/2020)
+-----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Upgrade to pex 2.1.6. (#9255)
+  `PR #9255 <https://github.com/pantsbuild/pants/pull/9255>`_
+
+* Remove deprecated `pants.backend.python` files moved to `pants.python` (#9238)
+  `PR #9238 <https://github.com/pantsbuild/pants/pull/9238>`_
+
+* Remove deprecated `--target-spec-file` and `--positional-arg-file` in favor of `--spec-file` (#9237)
+  `PR #9237 <https://github.com/pantsbuild/pants/pull/9237>`_
+
+* Remove deprecated `--owner-of` in favor of file arguments (#9235)
+  `PR #9235 <https://github.com/pantsbuild/pants/pull/9235>`_
+
+* Remove deprecated `--dependencies-internal-only` and `--dependencies-external-only` (#9258)
+  `PR #9258 <https://github.com/pantsbuild/pants/pull/9258>`_
+
+* Remove deprecated `--fmt-skip` and `--lint-skip` (#9254)
+  `PR #9254 <https://github.com/pantsbuild/pants/pull/9254>`_
+
+* Revert removing `--test-fast` and delay the removal from 1.27.0.dev0 to 1.28.0.dev0 (#9251)
+  `PR #9251 <https://github.com/pantsbuild/pants/pull/9251>`_
+
+* Revert "Load the bootstrapped zinc compiler from the zinc server's classpath. (#8753)" (#9226)
+  `PR #9226 <https://github.com/pantsbuild/pants/pull/9226>`_
+
+* Stop defaulting to `--transitive` and remove the option when it no-ops (#9236)
+  `PR #9236 <https://github.com/pantsbuild/pants/pull/9236>`_
+
+* Remove `--glob-expansion-failure` and deprecate `--files-not-found-behavior=ignore` (#9242)
+  `PR #9242 <https://github.com/pantsbuild/pants/pull/9242>`_
+
+* Remove `--test-fast` to always have better test caching (#9241)
+  `PR #9241 <https://github.com/pantsbuild/pants/pull/9241>`_
+
+* Remove deprecated `dict_option`, `list_option`, and `target_list_option` (#9243)
+  `PR #9243 <https://github.com/pantsbuild/pants/pull/9243>`_
+
+* Deprecate `--build-file-imports` and remove its deprecated value `allow` (#9234)
+  `PR #9234 <https://github.com/pantsbuild/pants/pull/9234>`_
+
+* Remove deprecated `fmt` and `lint` task options moved to subsystem options (#9246)
+  `PR #9246 <https://github.com/pantsbuild/pants/pull/9246>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Add support for `--dependencies-type=3rdparty` to `dependencies2` (#9225)
+  `PR #9225 <https://github.com/pantsbuild/pants/pull/9225>`_
+
+* pants build committing publish data for push of org.pantsbuild#zinc-compiler_2.12;0.0.20["ci skip"]
+
+* first implement v1 of robot impl (#8793)
```

Comment:
Can you please copy over the original PR title? I think Danny had intended to use the PR title when merging. I’m on my phone but the original mentioned ipex.